community-managed-token/program/src/instruction.rs

```diff
@@ -238,6 +238,7 @@ pub fn create_burn_instruction(
 pub fn create_close_account_instruction(
```

Comment:
we should just have a new pub fn `create_close_account_with_destination_instruction`